apps/docs/next-sitemap.config.js

```diff
@@ -0,0 +1,5 @@
+/** @type {import('next-sitemap').IConfig} */
+module.exports = {
+  siteUrl: "https://puckeditor.com",
+  generateIndexSitemap: false,
+};
```

Comment:
How can I test this? When I visit `/sitemap.xml` I get a 404.

Reply:
Reployed and it's back 🤷🏻‍♂️

https://puck-docs-git-seo-bits-measured.vercel.app/sitemap.xml

Reply:
I think when I force pushed 3 days ago – only to change commit msgs to `docs:` – that must have wiped the sitemap out (maybe by not doing a full build/postbuild).